include/bout/invertable_operator.hxx

```diff
@@ -134,7 +134,8 @@ public:
       : operatorFunction(func), preconditionerFunction(func),
         opt(optIn == nullptr ? optIn
```

Comment:
Not related to your code but have we (probably I) got this ternary the wrong way around?

Reply:
Oops, good spot!